controller/soap_web_service.rst

```diff
@@ -170,3 +169,4 @@ An example WSDL is below.
 .. _`NuSOAP`: https://sourceforge.net/projects/nusoap
```

Comment:
Needed?

Reply:
At the beginning of the document there is a short note about the available PHP implementations. NuSOAP is given as an example.